packages/react-components/src/components/trace-context-component.tsx

```diff
@@ -357,12 +358,14 @@ export class TraceContextComponent extends React.Component<TraceContextProps, Tr
                             return <XYOutputComponent key={output.id} {...outputProps} />;
                         case 'TABLE':
                             return <TableOutputComponent key={output.id} {...outputProps} />;
+                        case 'DATA_TREE':
+                            return <DataTreeOutputComponent key={output.id} {...outputProps} />;
                         default:
                             return <NullOutputComponent key={output.id} {...outputProps} />;
                     }
                 })}
             </ResponsiveGridLayout>
-            {(outputs.length > 1 || outputs[0].type !== 'TABLE') &&
+            {(outputs.length > 1 || outputs[0].type !== 'TABLE' && outputs[0].type !== 'DATA_TREE') &&
```

Comment:
It shouldn't add the TimeNavigatorComponent unconditionally where there are 2 or more outputs. It should do some fancy loop lookup to see if at least one of the outputs has a time axis. I wonder if it would be better to do a positive check for timegraph/xy instead of a negative check for table/datatree?

Also this condition should be applied to determine if TimeAxisComponent should be created. I would put the comment above but you know, github.